src/app/pages/system/tunable/tunable-form/tunable-form.component.ts

```diff
@@ -36,50 +36,60 @@ export class TunableFormComponent {
       type: 'input',
       name: 'tun_var',
       placeholder: T('Variable'),
-      tooltip: T('The name of the sysctl or driver to load.'),
+      tooltip: T('Enter the name of the loader, sysctl, or rc.conf\
+                  variable to configure.<br>\
+                  <b>loader</b>: Tunables that can be set only at boot\
```

Comment:
It's not when they are set but when they take effect.  Also, this switches back and forth to telling when a setting takes effect to what it does.  It should really do both.  We should avoid hard-coded line breaks in tooltips, and also bold.  Is it loader, or loader.conf?
"<i>loader</i> tunables are for... and only take effect after a restart. <i>rc.conf</i> tunables are for enabling system services and daemons, and only take effect after a reboot. <i>sysctl</i> tunables are for... and generally take effect immediately."

"any time" is two words.

Reply:
The intended use of set was when they take effect. Rereading the tooltip I can see the potential confusion especially with how we use set/unset for checkboxes. I'll reword this using your suggestions.

The bold was added before we discussed to eradicating the bolditis. I'll remove the bold. 

loader and sysctl are the variables, sysctl.conf and loader.conf are where the variables are set. I don't think it makes sense to talk about sysctl.conf or loader.conf here as the user isn't adding a new sysctl.conf or loader.conf, they are adding a new sysctl or loader variable and value. Also they are adding the variable using the webui, not to the respective conf. 
rc.conf is a tad different as one isn't adding an "rc", they enable/disable services and options to services. Changing rc.conf to "system service or system service options" would not only be confusing it's a tad wordy. I was going to say link to the user guide but we don't talk about rc.conf specifically there so I'd say link to the FreeBSD man page here. rc.conf is also well known and established. I'd argue if a user doesn't know what rc.conf is they maybe shouldn't be editing this based on the tooltip description. If enabling/disabling a service, or service options, calling this rc.conf would make the section easier to find and also how the options to set here would be explained if one was searching the internet for assistance.

sysctl, loader, and rc.conf are the options that were already in the tooltip and used in the user guide as well.

I know you don't like the use of hard-coded line breaks. I used them purposely here to visually separate the info. Without the line breaks the info ran together and was hard to quickly see there are multiple terms being explained. I'm not using the linebreak to force wrapping of a sentence but to visually separate the distinct items.

Reply:
when the variable is applied is specified/described in the Type tooltip.

Reply:
Updated this section to be more paragraph like rather than item:description.